config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "apiEndPoint": "http://localhost:8001",
+  "apiEndPoint": "http://localhost:3000/server",
```

Comment:
The default server entpoint is the same port as the client, but with a path prefix /server (kinda how i setup mine in production).